expression/schema.go

```diff
@@ -179,6 +179,16 @@ func MergeSchema(lSchema, rSchema *Schema) *Schema {
 	return ret
 }
 
+// GetUsedList shows whether each column in schema is contained in usedCols.
+func GetUsedList(usedCols []*Column, schema *Schema) []bool {
```

Comment:
Why do we move the function to package expression?

Reply:
It will be used by executor in future PR.